backend/manager/modules/dal/src/main/java/org/ovirt/engine/core/dao/StorageServerConnectionDaoImpl.java

```diff
@@ -116,6 +116,18 @@ public List<StorageServerConnections> getAllForConnection(
                         .addValue("username", connection.getUserName()));
     }
 
+    @Override
+    public List<StorageServerConnections> getStorageConnectionsByConnectionPortAndIqn(
```

Comment:
Using the same name (i.e., `getStorageConnectionsByConnectionPortAndIqn`) for method name & query could help in (case-insensitive) search.
Or this was on purpose?